### PR TITLE
016/master

### PR DESCRIPTION
**Pull request by gaand**

### CODE REVIEW

File: scripts/clinic/010_insert_into_patients.sql [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+  ('Rollins', 'Henry', 'm', 69, 210, '1961-02-13')
+;
+*/
+
+/* How to do it with selects.  Note the cast for born_on.
+*/
+INSERT INTO patients(family_name,given_name,gender,height,weight,born_on)
+  SELECT 'Jett', 'Joan', 'f', 65, 120, DATE '1958-09-22'
+  UNION
+  SELECT 'Rollins', 'Henry', 'm', 69, 210, '1961-02-13'
```
Comment (raq929) [inline scripts/clinic/010_insert_into_patients.sql:14]: What do you mean by 'cast'?

File: scripts/clinic/010_insert_into_patients.sql [hunk 2]
```diff
@@ -1,5 +1,15 @@
 -- insert data about patients
-INSERT INTO patients(surname,given_name,gender,height,weight,born_on) VALUES
-  ('Rollins', 'Henry', 'm', 69, 210, '1961-02-13'),
-  ('Smith', 'Patti', 'f', 68, 110, '1947-12-30');
-  
+/* Pretty standard approach
+INSERT INTO patients(family_name,given_name,gender,height,weight,born_on) VALUES
+  ('Jett', 'Joan', 'f', 65, 120, DATE '1958-09-22'),
+  ('Rollins', 'Henry', 'm', 69, 210, '1961-02-13')
+;
```
Comment (raq929) [inline scripts/clinic/010_insert_into_patients.sql:6]: Why do you specify date on line 4 but not on line 5?

**Review by raq929 — APPROVED** [review]
I have questions, but they don't prevent the merge. 😛

**Comment by raq929** [comment]
If updating from previous solution branches will de facto update from template, I think we should note that in the commit message.

File: scripts/clinic/010_insert_into_patients.sql (hunk 1 above)
Comment (raq929) [reply scripts/clinic/010_insert_into_patients.sql:14]: (In the comment. This question is on the wrong line.)

Comment (gaand) [reply scripts/clinic/010_insert_into_patients.sql:14]: A "cast" is a coercion from one data type to another.   In the above, string to date.

**Comment by gaand** [comment]
I believe that's potentially redundant and not necessarily true.

**Comment by raq929** [comment]
Good point. I guess what I'm saying is I want some indication in the commit or the PR message that there's been an update from template. Or not, if it's not necessary.

**Pull request merged**